website/pages/faq-es.md

```diff
@@ -172,4 +172,4 @@ Del mismo modo, Clefable no puede conocer Soft-Boiled y usar su habilidad oculta
 
 Escribe `/rank` y en el cuadro habrá un enlace a 'Reset W / L'. Hacer click en dicho enlace te permitirá reiniciar tu ratio. Ten en cuenta que tus otras clasificaciones no se verán afectadas; solo cambiará tu ratio de victorias / derrotas. Para cambiar estas otras clasificaciones, puedes comenzar de cero con una nueva cuenta.
 
-<small>(Originalmente por [Relados](https://www.smogon.com/forums/members/relados.148715/) y [Zarel](https://www.smogon.com/forums/members/zarel.102803/); actualizado por [Blitzamirin](https://www.smogon.com/forums/members/blitzamirin.101585/), [Kalalokki](https://www.smogon.com/forums/members/kalalokki.199661/), Kasumi, [shnen](https://www.smogon.com/forums/members/shnen.107178/), Sobi, [Finland](https://www.smogon.com/forums/members/finland.517429/), [Vacate](https://www.smogon.com/forums/members/vacate.189371/), [verbatim](https://www.smogon.com/forums/members/verbatim.93103/), [Zodiax](https://www.smogon.com/forums/members/zodiax.202547/) e [Iyarito](https://www.smogon.com/forums/members/iyarito.219578/); convertido a Markdown para el sitio de PS! por [Annika](https://www.smogon.com/forums/members/annika.434112/))</small>
\ No newline at end of file
+<small>(Originalmente por [Relados](https://www.smogon.com/forums/members/relados.148715/) y [Zarel](https://www.smogon.com/forums/members/zarel.102803/); actualizado por [Blitzamirin](https://www.smogon.com/forums/members/blitzamirin.101585/), [Kalalokki](https://www.smogon.com/forums/members/kalalokki.199661/), Kasumi, [shnen](https://www.smogon.com/forums/members/shnen.107178/), Sobi, [Finland](https://www.smogon.com/forums/members/finland.517429/), [Vacate](https://www.smogon.com/forums/members/vacate.189371/), [verbatim](https://www.smogon.com/forums/members/verbatim.93103/), [Zodiax](https://www.smogon.com/forums/members/zodiax.202547/) e [Iyarito](https://www.smogon.com/forums/members/iyarito.219578/); convertido a Markdown para el sitio de PS! por [Annika](https://www.smogon.com/forums/members/annika.434112/))</small>
```

Comment:
I do not understand why this is changing. I hard reset this file several times and only changed the one line, and even made the original commit without the line and changed it in the web editor. I don't get this.